web/viewer-geckoview.js

```diff
@@ -71,10 +69,9 @@ function webViewerLoad() {
       document.head.append(link);
     }
 
-    Promise.all([
-      import("pdfjs-web/genericcom.js"),
-      import("pdfjs-web/pdf_print_service.js"),
-    ]).then(function ([genericCom, pdfPrintService]) {
+    Promise.all([import("pdfjs-web/genericcom.js")]).then(function ([
+      genericCom,
+    ]) {
```

Comment:
Now that you only import one file here, this can be simplified as follows:
```suggestion
    import("pdfjs-web/genericcom.js").then(function (genericCom) {
```